ui/src/app/lib/atlasmap-data-mapper/models/mapping.model.ts

```diff
@@ -306,6 +306,16 @@ export class MappingModel {
     return paths;
   }
 
+  hasCollectionField(isSource: boolean): boolean {
+    const fields: Field[] = this.getFields(isSource);
+    for (const field of fields) {
+      if (field.isCollection) {
```

Comment:
I think we have to check if any of ancesters is in collection as well - and I have seen it in some class already implemented. Let me check...

Reply:
Yep, it's `Field.isInCollection()` method

Reply:
Ahh - missed that - will fix...